recipes/ngmlr/mapq.patch

```diff
@@ -0,0 +1,15 @@
+Avoid MAPQ = -2147483648, which causes samtools >= 1.10 to fail.
+Based on https://github.com/philres/ngmlr/pull/96
+
+diff --git a/src/AlignmentBuffer.cpp b/src/AlignmentBuffer.cpp
+index 26130d1..33597ec 100644
+--- a/src/AlignmentBuffer.cpp
++++ b/src/AlignmentBuffer.cpp
+@@ -1888,6 +1888,7 @@ int AlignmentBuffer::computeMappingQuality(Align const & alignment, int readLeng
+ 		mqSum += results[j].value;
+ 		mqCount += 1;
+ 	}
++	if (mqCount == 0) return 255; // MAPQ "not available", but at least a valid value

```

Comment:
Can you discuss this version in the referenced PR? I will ping Michael to merge it.
Thanks.

Reply:
Will do.